### PR TITLE
Notifications by time

### PR DESCRIPTION
**Pull request by elivina997**

### CODE REVIEW

File: app/src/main/java/com/univer/covid19help/MainFragment.kt [hunk 1]
```diff
@@ -248,31 +250,53 @@ class MainFragment : Fragment() {
                 when(TimeDiff.diff(Date(), calendar.time)) {
                     TimeDiff.Diff.TOO_EARLY -> {
                         Log.i(TAG,"onTimer [$state] -> TOO_EARLY")
+                        binding.txtNotify.text = resources.getString(R.string.txtEarly)
```
Comment (SergeyPjatkin) [inline app/src/main/java/com/univer/covid19help/MainFragment.kt:253]: we do not need to notify about too early. just keep it blank. or do nothing

File: app/src/main/java/com/univer/covid19help/MainFragment.kt [hunk 2]
```diff
@@ -248,31 +250,53 @@ class MainFragment : Fragment() {
                 when(TimeDiff.diff(Date(), calendar.time)) {
                     TimeDiff.Diff.TOO_EARLY -> {
                         Log.i(TAG,"onTimer [$state] -> TOO_EARLY")
+                        binding.txtNotify.text = resources.getString(R.string.txtEarly)
+
+                    }
+                    TimeDiff.Diff.LESS_THAN_DAY -> {
+                        Log.i(TAG,"onTimer [$state] -> LESS_THAN_DAY")
+                        SystemAndroid.notify("24 hours")
```
Comment (SergeyPjatkin) [inline app/src/main/java/com/univer/covid19help/MainFragment.kt:258]: do not use hard coded strings for user interface. create resources.

File: app/src/main/java/com/univer/covid19help/MainFragment.kt [hunk 3]
```diff
@@ -248,31 +250,53 @@ class MainFragment : Fragment() {
...
                         SystemAndroid.notify("NOW")
                         SystemAndroid.notifyToast("NOW")
+                        binding.txtNotify.text = resources.getString(R.string.txtNow)
                     }
                     TimeDiff.Diff.NONE -> {
                         Log.d(TAG,"onTimer [$state] -> NONE")
                     }
                     TimeDiff.Diff.TOO_LATE -> {
                         Log.i(TAG,"onTimer [$state] -> TOO_LATE")
+                        SystemAndroid.notify("Late")
```
Comment (SergeyPjatkin) [inline app/src/main/java/com/univer/covid19help/MainFragment.kt:297]: i think we do not need to ping user about "too late". consider do nothing here

File: app/src/main/java/com/univer/covid19help/MainFragment.kt (hunk 1 above)
Comment (elivina997) [reply app/src/main/java/com/univer/covid19help/MainFragment.kt]: Done

File: app/src/main/java/com/univer/covid19help/MainFragment.kt (hunk 2 above)
Comment (elivina997) [reply app/src/main/java/com/univer/covid19help/MainFragment.kt]: Done

File: app/src/main/java/com/univer/covid19help/MainFragment.kt (hunk 3 above)
Comment (elivina997) [reply app/src/main/java/com/univer/covid19help/MainFragment.kt]: Done

**Review by SergeyPjatkin — APPROVED** [review]

**Pull request merged**